pkgs/by-name/ju/junest/package.nix

```
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, makeWrapper, wget }:
+
+stdenv.mkDerivation rec {
+  version = "7.4.8";
+  pname = "junest";
+  src = fetchFromGitHub {
+    owner = "fsquillace";
+    repo = "junest";
+    rev = "refs/tags/${version}";
+    hash = "sha256-9yrQ721fHUxXEZ0mh27SB8yoUH67ltOktUq3kr4qrBc=";
+  };
+
+  dontBuild = true;
+
+  buildInputs = [
+    wget
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/lib
+    cp -r $src/bin/ $out/
+    cp -r $src/lib/ $out/
+    substituteInPlace $out/lib/core/common.sh --replace "wget" "${wget}/bin/wget"
```

```suggestion
    substituteInPlace $out/lib/core/common.sh --replace-fail "wget" "${wget}/bin/wget"
```

replace is deprecated